PYME/Analysis/points/spherical_harmonics.py

```diff
@@ -374,7 +374,7 @@ def set_fitting_points(self, x, y, z):
     def _scale_fitting_points(self):
         self.standard_deviations, self.principal_axes = coordinate_tools.find_principle_axes(self.x_c, self.y_c, self.z_c,
                                                                                              sample_fraction=self.sampling_fraction)
-        self.scaling_factors = np.max(self.standard_deviations) / (self.standard_deviations)
+        self.scaling_factors = (self.standard_deviations) / np.max(self.standard_deviations)
```

Comment:
@barentine - this is the change I mentioned to the scaling. It inverts the scaling factors, but I think this is correct as we multiply the co-ordinates by them (i.e. we should blow up our z when fitting so that it has a similar range to x and y). 

Before this change, it would seem that we double-down and make the smallest axis even smaller.

Reply:
argh, no I was wrong in my reading. Initial code was correct. I knew it would be good to get another view on this.